assignment-8-flipkart-page-1/index.html

```diff
@@ -0,0 +1,799 @@
+                    </div>
+                    <div class="s1">
+                      <p>1,81059</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="review">
+                <a class="rating1" href="">9,002 Ratings & 708 Reviews</a>
+              </div>
+            </div>
+            <nav class="navigation3">
+              <ul>
+                <li><a href="">6 GB RAM | 128 GB ROM | Expandable Upto 128 GB </a></li>
+                <li><a href="">16.0cm (6.3 inch) Full HD+Display </a></li>
+                <li><a href="">48MP + 8MP | 13MP Front Camera </a></li>
+                <li><a href="">4000 mah Li-polymer Battery</a></li>
+                <li><a href="">Splash Proof - Protected by P2i</a></li>
+                <li><a href="">Quick Charge 4.0 Support</a></li>
+                <li><a href="">Brand Warranty of 1 Year Available for Mobile and 6 Months for Accessories</a></li>
+              </ul>
+            </nav>
+          </div>
+
+          <div class="price">
+            <div class="rate">
+              <div>
+                <a class="rate1" href="">₹16,599</a>
+              </div>
+              <div class="rate2">
+                <a href=""><img src="img/assured.png" width="65"></a>
+              </div>
+            </div>
+
+            <div class="emi">
+              <p>No Cost EMI</p>
+            </div>
+          </div>
+
+        </div>
+      </div>
+    </section>
+  </main>
+
+  <footer>
+    <nav class="footerNav">
+      <ul>
+
+        <li>
+          <div class="miniContainer">
+            <div class="footerimg">
+              <a href=""><img src="img/f1.PNG"></a>
+            </div>
+            <div class="footertext">
+              <a href="">Sell on Flipkart</a>
+            </div>
+          </div>
+        </li>
+
+        <li>
+          <div class="miniContainer">
+            <div class="footerimg">
+              <a href=""><img src="img/f2.PNG"></a>
+            </div>
+            <div class="footertext">
+              <a href="">Advertise</a>
+            </div>
+          </div>
+        </li>
+
+        <li>
+          <div class="miniContainer">
+            <div class="footerimg">
+              <a href=""><img src="img/f3.PNG"></a>
+            </div>
+            <div class="footertext">
+              <a href="">Gift Cards</a>
+            </div>
+          </div>
+        </li>
+
+        <li>
+          <div class="miniContainer">
+            <div class="footerimg">
+              <a href=""><img src="img/f4.PNG"></a>
+            </div>
+            <div class="footertext">
+              <a href="">Help Center</a>
+            </div>
+          </div>
+        </li>
+
+        <li class="copyright"><a href="">&#169 2007-2020 flipkart.com</a></li>
+        <li class="card"><a href=""><img src="img/card.jpeg"></a></li>
+
+      </ul>
+    </nav>
+  </footer>
+</body>
+
+</html>
```

Comment:
always leave an empty line at the end of file, it is a standard coding practice.